bin/verifyWeeSchema.js

```diff
@@ -0,0 +1,65 @@
+
+/**
+  * given a propose wee interface schama.json, lint and check it for
```

Comment:
@aswan Is there a standardized/documented process to validate WEE API `schema.json`?